cmd/jb/main.go

```diff
@@ -100,6 +105,120 @@ func initCommand() int {
 	return 0
 }
 
+func parseDepedency(urlString string) *spec.Dependency {
```

Comment:
Between shadowing the net/url module and the "stutter" from `urlString string` I opted for the stutter.